internal/cmd/tui.go

```diff
@@ -339,6 +341,41 @@ func (m tuiModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 
 			return m, tea.Quit
+
+		case "ctrl+w", "w":
+			indexes, err := fuzzyfinder.FindMulti(
+				m.blocks,
+				func(i int) string {
+					return m.blocks[i].GetBlock().Name()
+				},
+				fuzzyfinder.WithPreviewWindow(func(i, w, h int) string {
+					codeLines := m.blocks[i].GetBlock().Lines()
```

Comment:
Let's also add the markdown file here - can be on the first line, surrounded with `---`, like:

```
--- FILENAME.md ---

...
```